Multisig/Cross-layer/App.swift

```diff
@@ -25,13 +25,17 @@ class App {
     // Data Layer
 
     // Services    
-    var safeTransactionService = SafeTransactionService(url: URL(string: "https://safe-transaction.rinkeby.gnosis.io")!,
+    var safeTransactionService = SafeTransactionService(url: URL(string: "https://safe-transaction.staging.gnosisdev.com/")!,
     logger: LogService.shared)
 
+    var safeRelayService = SafeRelayService(url: URL(string: "https://safe-relay.staging.gnosisdev.com")!, logger: LogService.shared)
```

Comment:
Did we add to the list of services in the app settings advanced section?

Reply:
now I did

Reply:
no, we didn't, this is a temporary solution, though